kamel-image/src/commonMain/kotlin/io/kamel/image/LazyPainterResource.kt

```diff
@@ -52,13 +54,22 @@ public inline fun <I : Any> lazyPainterResource(
             .build()
     }
 
+    val cachedOutput = remember(key) {

```

Comment:
In the ImageLoading file could you just make a new public function called something like `loadCachedResource` that returns the cached resource or null.